### PR TITLE
[excel] (Range) Conceptual docs for group and ungroup

### PR DESCRIPTION
**Pull request by AlexJerabek**
These are the corresponding conceptual docs for [this new Script Lab sample](https://github.com/OfficeDev/office-js-snippets/pull/352). It explains outline behavior and highlights some subtleties in the API.

This PR also does a small consistency scrub on the Ranges (advanced) article.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[e007996](https://github.com/OfficeDev/office-js-docs-pr/commits/e00799688028eabf346688559fc3d54b2d02259c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-ranges-advanced.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Outline/docs/excel/excel-add-ins-ranges-advanced.md) | :white_check_mark:Succeeded | |
[docs/images/excel-outline.png](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Outline/docs/images/excel-outline.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909182144595315-1310%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2VmIpkVeu4WTHhd3IUpHmckgYHoRhQ%2B9kSB%2FVHOdcCE%3D&st=2019-09-18T21%3A41%3A05Z&se=2019-10-19T21%3A46%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/excel-add-ins-ranges-advanced.md [hunk 1]
```diff
@@ -264,6 +264,48 @@ Excel.run(async (context) => {
...
 ![Data in Excel after range’s remove duplicates method has been run](../images/excel-ranges-remove-duplicates-after.png)
 
+## Group data for an outline
+
+> [!NOTE]
+> The outline APIs for grouping rows and columns are currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+
+Rows or columns of a range can be grouped together to create an [outline](https://support.office.com/article/Outline-group-data-in-a-worksheet-08CE98C4-0063-4D42-8AC7-8278C49E9AFF). These groups can be collapsed and expanded to hide and show the corresponding cells. This makes quick analysis of top-line data easier. Use [Range.group](/javascript/api/excel/excel.range#group-groupoption-) to make these outline groups.
+
+An outline has a hierarchy. Smaller groups nest under larger groups. This allows the outline to be viewed at different levels. Changing the visible outline level can be done programmatically through the [Range.showOutlineLevels](/javascript/api/excel/excel.range#showOutlineLevels-rowLevels--columnLevels-) method. Note that Excel only supports eight levels of outline groups.
```
Comment (kbrandl) [inline docs/excel/excel-add-ins-ranges-advanced.md]:
```suggestion
An outline can have a hierarchy, where smaller groups are nested under larger groups. This allows the outline to be viewed at different levels. Changing the visible outline level can be done programmatically through the [Range.showOutlineLevels](/javascript/api/excel/excel.range#showOutlineLevels-rowLevels--columnLevels-) method. Note that Excel only supports eight levels of outline groups.
```

Comment (kbrandl) [inline docs/excel/excel-add-ins-ranges-advanced.md]: The target of the link `[Range.showOutlineLevels](/javascript/api/excel/excel.range#showOutlineLevels-rowLevels--columnLevels-)` doesn't seem to exist yet -- assuming it just hasn't made its way into the ref docs yet, but will soon?

File: docs/excel/excel-add-ins-ranges-advanced.md [hunk 2]
```diff
@@ -264,6 +264,48 @@ Excel.run(async (context) => {
...
+    sheet.getRange("H:K").group(Excel.GroupOption.byColumns);
+    sheet.getRange("M:P").group(Excel.GroupOption.byColumns);
+    return context.sync();
+}).catch(errorHandlerFunction);
+
+```
+
+![A range with a two-level, two-dimension outline](../images/excel-outline.png)
+
+To ungroup a row or column group, use the [Range.upgroup](/javascript/api/excel/excel.range#ungroup-groupoption-) method. This removes the outer-most level from the outline. If multiple groups of the same row or column type are at the same level within the specified range, all those groups are ungrouped.
```
Comment (kbrandl) [inline docs/excel/excel-add-ins-ranges-advanced.md]:
```suggestion
To ungroup a row or column group, use the [Range.upgroup](/javascript/api/excel/excel.range#ungroup-groupoption-) method. This removes the outermost level from the outline. If multiple groups of the same row or column type are at the same level within the specified range, all of those groups are ungrouped.
```

File: docs/excel/excel-add-ins-ranges-advanced.md (hunk 1 above)
Comment (kbrandl) [reply docs/excel/excel-add-ins-ranges-advanced.md]: No worries! I knew you were probably accounting for this already :)

**Comment by AlexJerabek** [comment]
This is dependent on https://github.com/OfficeDev/office-js-docs-reference/pull/516

**Pull request merged**